data/weiji.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Weiji",
+  "url": "https://weiji.it",
+  "career_page_url": "https://weiji.it/contatti/",
+  "type": "Product",
```

Comment:
Hi @robertok-weiji, could not find any evidence that this company hires remotely nor that it's a product company. Can you please elaborate a bit on these two topics? Thanks.

Reply:
Hi @edoardocostantinidev it is not explicitly stated in the website, the company has its physical headquarter in Turin, but offers the possibility of partial or full remote work since 2019. Here https://weiji.it/2020/09/11/cose-lo-shbook/ is described the nature of our product, dedicated to digital books publishing companies. Thanks.

Reply:
thanks a lot for your contribution! Approved 👍🏻